src/styles/index.scss

```diff
@@ -1,3 +1,11 @@
+@import './utils/variables';
+@import './utils/mixins';
+@import 'blocks/card';
+@import 'blocks/stars';
```

Comment:
Ensure that the file 'blocks/stars' exists and is correctly configured. Missing files will lead to compilation errors.